ShowdownAI.Middleware/ShowdownAI.Middleware/Models/TargetType.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ShowdownAI.Middleware.Models
+{
+    public enum TargetType
+    {
+        None,
+
+        // single-target
+        normal,
+        any,
+        adjacentAlly,
+        adjacentFoe,
+        adjacentAllyOrSelf,
+
+        // single-target, automatic
+        self,
+        randomNormal,
+
+        // spread
+	    allAdjacent,
```

Comment:
Formatting. In Visual Studio you can press Ctrl+K then Ctrl+D and it will format the file for you

Reply:
I actually found a C# linter that I run to format it and check for any issues. I set up a `pre-commit` file locally with it and some other commands to save me. 

I'm going to get around to setting up a GitHub Action at some point so it can shout at us if we mess up.